README.md

```diff
@@ -193,6 +213,32 @@ everything else from the plugin:
 
 For convenience, you'll find those configuration blocks in the example configuration file [`10-docker.conf`](https://github.com/signalfx/integrations/blob/master/collectd-docker/10-docker.conf).
 
+## How to send metrics about resource allocation
+
+In addition to the above metrics if you are also looking to get few advanced metrics on resource allocation, add the following snippet to the filter.
```

Comment:
consider
"Additional resource allocation metrics can be gathered by adding the following snippet to the plugin's filter configuration."

Reply:
"If a filter has been configured, then additional resource allocation metrics can be gathered by adding the following snippet to the plugin's filter configuration."

Please use this phrasing ^